Nette/Application/Routers/Route.php

```diff
@@ -251,13 +253,15 @@ public function match(Nette\Http\IRequest $httpRequest)
 			unset($params[self::PRESENTER_KEY]);
 		}
 
+		$this->httpRequestSecured = $httpRequest->isSecured();
```

Comment:
Please do not save the state here, it's not neccessary and makes it harder to test router.